### DISCUSSION TITLE
How does this work with domain from Cloudflare?

### DISCUSSION THREAD

**Discussion by socketopp** [digitalocean/supabase-on-do#32] [Q&A]
Hi,

I bought my domain at Cloudflare and trying to set this all up.

I have followed the follwing pre-requisite

- A Domain you own [added to DigitalOceans' Domain section](https://docs.digitalocean.com/products/networking/dns/how-to/add-domains/) and the nameservers in your chosen domain registrar pointed towards DigitalOceans' own NS records([docs](https://docs.digitalocean.com/tutorials/dns-registrars/)).

My records in Cloudflare look identical to the ones found in the Domain in DO currently. 
However the URL `supabase.[my-domain].com` does not work. 

I also try to access the droplets IP address but that fails just like the domain. Any ideas? 

Is there anything else I should be doing?

**Comment by timothymamo** [comment]
Hey @socketopp, have you followed the following [advice](https://github.com/digitalocean/supabase-on-do/discussions/17#discussioncomment-5171161)? This should give you an indication of what you'd need to do if the domain is with CloudFlare.
